extensions/gdx-setup/src/com/badlogic/gdx/setup/DependencyBank.java

```diff
@@ -42,14 +42,14 @@
 	//Project plugins
 	static String gwtPluginImport = "org.wisepersist:gwt-gradle-plugin:1.0.13";
 	static String grettyPluginImport = "org.gretty:gretty:3.0.2";
-	static String androidPluginImport = "com.android.tools.build:gradle:3.4.1";
+	static String androidPluginImport = "com.android.tools.build:gradle:3.4.3";
```

Comment:
Just as a note, there's already 4.0.2 available.

Reply:
I wasn't sure if all versions of AS were compatible yet. I didn't want to invite any unnecessary breakage by changing a major version.